app/src/main/java/io/homeassistant/companion/android/settings/qs/views/ManageTilesView.kt

```diff
@@ -0,0 +1,144 @@
+package io.homeassistant.companion.android.settings.qs.views
+
+import android.graphics.PorterDuff
+import android.widget.Toast
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.material.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringArrayResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.core.graphics.drawable.DrawableCompat
+import androidx.core.graphics.drawable.toBitmap
+import androidx.fragment.app.FragmentManager
+import com.maltaisn.icondialog.IconDialog
+import com.maltaisn.icondialog.pack.IconPack
+import com.maltaisn.icondialog.pack.IconPackLoader
+import com.maltaisn.iconpack.mdi.createMaterialDesignIconPack
+import io.homeassistant.companion.android.common.R
+import io.homeassistant.companion.android.database.AppDatabase
+import io.homeassistant.companion.android.database.qs.TileEntity
+import io.homeassistant.companion.android.settings.qs.ManageTilesViewModel
+
+private lateinit var iconPack: IconPack
+
+@Composable
+fun ManageTilesView(
+    viewModel: ManageTilesViewModel,
+    iconDialog: IconDialog,
+    childFragment: FragmentManager
+) {
+    val context = LocalContext.current
+    var expandedTile by remember { mutableStateOf(false) }
+    var expandedEntity by remember { mutableStateOf(false) }
+    Column(modifier = Modifier.padding(20.dp)) {
+        TextButton(onClick = { expandedTile = true }) {

```

Comment:
I don't think making everything a large button is a good idea because 60% (or more, depending on how you calculate it) of the screen is now a button. Suggestion: use text with an outlined button like for the Wear OS template tile refresh interval setting on a phone (also for icon/entity selection):
![image](https://user-images.githubusercontent.com/8148535/150785796-91307c11-5457-4b80-837f-e13462cb26f4.png)

Reply:
On the row, [add `verticalAlignment = Alignment.CenterVertically`](https://github.com/home-assistant/android/blob/b7f6a7322cfc156f2854513bb5fa32d7fdbe89af/app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearTemplateTile.kt#L80)